packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -182,7 +185,8 @@ const ReviewVoteTableRow = (
             { post.reviewCount }
           </LWTooltip>
         </PostsItem2MetaInfo>
-        {getReviewPhase() === "REVIEWS" && <div className={classes.voteResults}>
+        {getReviewPhase() === "REVIEWS" && <div className={classes.votes}>
```

Comment:
I can't tell from here, but good to ensure this is an enum type, not just a string.

Reply:
It is.